framework/src/modules/chain/synchronizer/fast_chain_switching_mechanism.js

```diff
@@ -119,7 +119,7 @@ class FastChainSwitchingMechanism extends BaseSynchronizer {
 			return false;
 		}
 
-		const blockRound = this.slots.calcRound(receivedBlock.height);
+		const blockRound = this.rounds.calcRound(receivedBlock.height);
```

Comment:
```suggestion
		const blockRound = this.dpos.rounds.calcRound(receivedBlock.height);
```
and above `rounds` can be removed